### ISSUE TITLE
swiper faces for package-list-packages where "Archive" is filled

### ISSUE THREAD

**Issue by habamax** [abo-abo/swiper#492]
Why there is a different faces for searches with or without "Archive" filled in?

How can I find those faces and set them up?

See the screenshots:
Gray background where "melpa" is
<img width="1280" alt="achive is filled" src="https://cloud.githubusercontent.com/assets/234774/14923122/f40d610e-0e43-11e6-9bc0-23fc0b981d90.png">

Normal usual where there is no "melpa"
<img width="1280" alt="archive is not filled" src="https://cloud.githubusercontent.com/assets/234774/14923123/f40ddaee-0e43-11e6-8329-6478f8e4b455.png">

My problem is that in my black theme those Gray backgrounds are actually black. And I can not see selected item.
<img width="1280" alt="my theme" src="https://cloud.githubusercontent.com/assets/234774/14923192/7610f6a2-0e44-11e6-89d7-df4f63f6019a.png">

**Comment by abo-abo** [comment]
I guess the theme you're using in the first screenshot is leuven. I can't reproduce the issue on Emacs 25. What's your version?

**Comment by habamax** [comment]
Yes it is Leuven, version is from homebrew, I guess it is 24.5 (cannot check it at the moment)

**Comment by habamax** [comment]
M-x version

GNU Emacs 24.5.1 (x86_64-apple-darwin14.5.0, NS apple-appkit-1348.17)
